sormas-api/src/main/java/de/symeda/sormas/api/event/EventParticipantCriteria.java

```diff
@@ -13,12 +13,14 @@
 	public static final String BIRTHDATE_YYYY = "birthdateYYYY";
 	public static final String BIRTHDATE_MM = "birthdateMM";
 	public static final String BIRTHDATE_DD = "birthdateDD";
+	public static final String CONTACT_COUNT_ONLY_WITH_SOURCE_CASE_IN_EVENT = "contactCountOnlyWithSourceCaseInEvent";
 
 	private EventReferenceDto event;
 	private String freeText;
 	private Integer birthdateYYYY;
 	private Integer birthdateMM;
 	private Integer birthdateDD;
+	private Boolean contactCountOnlyWithSourceCaseInEvent;
```

Comment:
I renamed it to onlyCountContactsWithSourceCaseInEvent to avoid this "should contain a number" presumption.
But as in the other comment could you elaborate on `Naming should be in line with #3314;` please ?

Reply:
#3314 changed the field name and caption "onlyContactsWithSourceCaseInEvent" to "onlyContactsSharingEventWithSourceCase". This change should be reflected in this pull request because we'll have inconsistent naming otherwise.

Reply:
As for https://github.com/hzi-braunschweig/SORMAS-Project/pull/3344#discussion_r519970032 those two aren't equivalent.
And so I don't think we should use the same name.

Here we do not restrain the contactCount by eventParticipant's contacts which share the same event with their source case, but eventParticipant's contacts whose source case is the eventParticipant's event.

Let's say we have :
    Contact1 (Person1) linked to Event1, and its source case is linked to Event1.
    Contact2 (Person1) linked to Event2, and its source case is linked to Event2.
    Contact3 (Person1) linked to Event3, and its source case is linked to Event4.

We expect a contactCount of 3 by default and 1 when filtered further.
But with the the mechanic induced by the `onlyContactsSharingEventWithSourceCase` name we'd have a contactCount of 3 by default and 2 when filtered further.